tox.ini

```diff
@@ -198,21 +199,21 @@ deps =
     mlmodel_sklearn-scikitlearn0101: scipy<1.11.0
     component_djangorestframework-djangorestframeworklatest: Django
     component_djangorestframework-djangorestframeworklatest: djangorestframework
-    component_flask_rest: flask
     component_flask_rest: flask-restful
     component_flask_rest: jinja2
     component_flask_rest: itsdangerous
     component_flask_rest-flaskrestxlatest: flask-restx
-    ; Pin Flask version until flask-restx is updated to support v3
-    component_flask_rest-flaskrestxlatest: flask<3.0
+    component_flask_rest-flaskrestxlatest: flask
+    ; flask-restx only supports Flask v3 after flask-restx v1.3.0
+    component_flask_rest-flaskrestx110: Flask<3.0
+    component_flask_rest-flaskrestx110: flask-restx<1.2
+    component_flask_rest-flaskrestx051: Flask<3.0
     component_flask_rest-flaskrestx051: flask-restx<1.0
     component_graphqlserver: graphql-server[sanic,flask]==3.0.0b5
     component_graphqlserver: sanic>20
     component_graphqlserver: Flask
     component_graphqlserver: markupsafe<2.1
     component_graphqlserver: jinja2<3.1
-    component_tastypie-{py37,py38,py39,py310,py311,py312,pypy38}-tastypie0143: django<3.0.1
-    component_tastypie-{py37,py38,py39,py310,py311,py312,pypy38}-tastypie0143: asgiref<3.7.1  # asgiref==3.7.1 only suppport Python 3.10+
```

Comment:
Are these supposed to be deleted? What was the problem with tastypie? Support window?

Reply:
Yep, those are supposed to be deleted (maybe I missed it in the python 3.12 PR).  That version is past New Relic's support window